docs/nx-cloud/private/ami-setup.md

```diff
@@ -0,0 +1,170 @@
+   - Create an [application load balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-application-load-balancer.html)
+   - You will need to create a certificate for your domain to assign to the LB
+   - And you will need to target your EC2 instance from the LB
+   - You should now have a permanent domain pointing to your NxCloud instance
+
+Once you have your NxCloud URL proceed to the below steps!
+
+### Configuring your NxCloud instance
+
+1. Create a new `myconfiguration.yaml` file with the below contents
+
+```yaml
+# This is all you need to get the baseline of your nx-cloud instance configured!
+
+# Set the external URL your instance is running on. This is the URL from the previous step
+nxCloudAppURL: 'https://nx-cloud.on.my-domain.ca'
+
+secret:
+  # set your initial admin password for logging into the app
+  adminPassword: 'correcthorsebatterystaple'
+```
+
+2. Apply the configuration:
+
+```bash
+scp -r ./myconfiguration.yaml nx-cloud@<your-instance-ip>:~/config/user/update.yaml
+```
+
+That's it! After a few minutes, you should be able to log-in with:
+
+- username: `admin`
+- password: `<the-password-you-set-above>`
+
+### Applying the license
+
+Once you log-in, you will see an organisation has been created for you.
+
+1. You can rename it or create a new organization.
+2. Navigate to your new organization's page and send us it's id
+   - It should look something like this: https://your-url.com/orgs/649f240f7fb955000c1fd10b/workspaces
+3. We will then give you a License Key which you can apply on your org's billing page
+
+### Connecting to your NxCloud instance
+
+In your Nx workspace, you can enable NxCloud by running:
+
+```bash
+NX_CLOUD_API="https://nx-cloud.on.my-domain.ca" npx nx connect
+```
+
+If it doesn't work, there might be an issue with unrecognized certificates on your instance. You can try running with:
+
+```bash
+NODE_TLS_REJECT_UNAUTHORIZED=0 NX_CLOUD_API="https://nx-cloud.on.my-domain.ca" npx nx connect
+```
+
+Although we have [a full guide here](https://github.com/nrwl/nx-cloud-helm/blob/main/PROXY-GUIDE.md#nxcloud-runner-proxy-issues) for dealing with self-signed certificates.
+
+### Advanced configuration and auth
+
+You can optionally enable authentication using your preferred SSO provider:
+
+- GitHub
+- Bitbucket
+- GitLab
+- SAML (Okta, Azure AD etc.)
+
+```yaml
+# This is all you need to get the baseline of your nx-cloud instance configured!
+
+# only use this if you'd like to use any of the newer NxCloud version from here: https://nx.dev/nx-cloud/reference/release-notes#docker-containers
+# global.imageTag: ''
+
+# Set the external URL your instance is running on
+nxCloudAppURL: 'https://nx-cloud.on.my-domain.ca'
+
+# Uncomment (along with github secrets below) to enable working with GitHub pull requests or github auth
+#github:
+#  auth:
+#    enabled: false
```

Comment:
I think we spoke about this before, but when showing config samples we should encourage the user to set both the secret and the explicit configuration option. While we do have logic that will ensure it is set or unset correctly based on the secrets provided, I would rather they not be relying on that by default in case we change this behaviour for some reason in the future
```suggestion
github:
  auth:
    enabled: true
```